api/internal/git/repospec.go

```diff
@@ -30,16 +30,16 @@ type RepoSpec struct {
 	// Host, e.g. https://github.com/
 	Host string
 
-	// orgRepo name (organization/repoName),
+	// RepoPath name (organization/repoName),
 	// e.g. kubernetes-sigs/kustomize
-	OrgRepo string
+	RepoPath string
 
-	// Dir where the orgRepo is cloned to.
+	// Dir where the RepoPath is cloned to.
```

Comment:
super nit: Technically, `Dir` is `where the repository is cloned to`. The path to the repository isn't actually cloned anywhere.